magit-todos.el

```diff
@@ -359,6 +359,11 @@ from the \"topic2\" branch, this option could be set to
 \"topic\"."
   :type 'string)
 
+(defcustom magit-todos-show-submodule-notes t
+  "Show git submodule todo notes."
```

Comment:
For consistency with other option docstrings, how about `Show submodule to-do list.`